worker/pb/api.pb.go

```diff
@@ -255,7 +255,7 @@ type StreamRequest struct {
 	unknownFields protoimpl.UnknownFields
 
 	WorkerId      string                 `protobuf:"bytes,1,opt,name=WorkerId,proto3" json:"WorkerId,omitempty"`
-	SourceType    string                 `protobuf:"bytes,2,opt,name=SourceType,proto3" json:"SourceType,omitempty"`
+	SourceType    string                 `protobuf:"bytes,2,opt,name=SourceMode,proto3" json:"SourceMode,omitempty"`
```

Comment:
This is a generated file and should not be edited. If you want to change the api, please edit `worker/api/api.proto` and rebuild the grpc stub (see makefile)

Why does this need to be changed though?